recipes-ros/image-transport-plugins/image-transport-plugins.inc

```diff
@@ -1,6 +1,6 @@
 SRC_URI = "https://github.com/ros-perception/${ROS_SPN}/archive/${PV}.tar.gz;downloadfilename=${ROS_SP}.tar.gz"
-SRC_URI[md5sum] = "537c8dca08fe8a936438252b3ee6cdc8"
-SRC_URI[sha256sum] = "2d9b10f1e3c22145d58ba922dcafc72b149bb4561e641fff17eddf1510294271"
+SRC_URI[md5sum] = "855253ebefbcf508d2af7dd3e8d5d15a"
```

Comment:
On kinetic-experimental-v3-alpha1, image-transport-plugins have already been
updated to version 1.9.5. So, this hunk does not apply to kinetic-experimental-v3-alpha1
anymore.